src/styles/components/_code-block.scss

```diff
@@ -0,0 +1,70 @@
+$breakpoint-tablet: 820px;
+
+.cmp-code-block {
+  width: calc(100% - 2.5rem); // width 100% - 1.25rem on either side; 20px/16
+  margin: 3.75rem auto; // top&bottom: 60px/16; left&right: auto
+  max-width: 100rem; // 1600px/16
+
+  @media only screen and (min-width: $breakpoint-tablet) {
+    width: 100%;
+    margin: auto;
+    padding: 3.75rem 8.5625rem; // top&bottom: 60px/16; left&right: 137px/16
+  }
+
+  &__heading {
+    font-size: 1.5625rem; // 25px/16
+    font-family: Viga, sans-serif;
+    line-height: 2.125rem; // 34px/16
+
+    &--arrow {
+      transform: rotate(-30deg);
+      max-width: 13%; // visually this looked correct
+      position: relative;
+      top: 2.5rem; // 24px/16
+      left: 1rem; // 16px/16
+    }
+
+    @media only screen and (min-width: $breakpoint-tablet) {
+      font-size: map.get($font-sizes, "md");
+      line-height: 2.5rem; // 40px/16
+    }
+  }
+
+  &__content {
+    padding: 1.875rem 1.25rem; // top&bottom: 30px/16; left&right: 20px/16
+    background-color: map.get($colors, "black");
+    color: map.get($colors, "white");
+    line-height: 1.75rem; // 28px/16
+    font-size: map.get($font-sizes, "xs");
+    font-family: Oxygen, sans-serif;
+    letter-spacing: 0.02em; // 2% in Figma
+    margin-top: 3rem;
+
+    @media only screen and (min-width: $breakpoint-tablet) {
+      background-color: map.get($colors, "white");
+      color: map.get($colors, "black");
+      border: 2px solid map.get($colors, "black");
+      padding: 2.5rem 3.75rem; // top&bottom: 40px/16; left&right: 60px/16
+      margin: 0;
+    }
+  }
+
+  pre {
+    margin: 0;
+    white-space: pre;
+    overflow-x: scroll;
+    max-width: 75ch;
```

Comment:
I'd omit the `max-width` here. It ends up arbitrarily cutting off the content before filling the whole box.

![Screenshot 2022-11-04 at 12 05 52 PM](https://user-images.githubusercontent.com/82047862/200045420-b525407b-49ca-4f8d-b61d-9a94f2276857.png)

Reply:
Gosh dang it, yup. Jon told me that yesterday when he reviewed it, and I must have missed it when I rebased.

Reply:
It's fixed in the latest commit: https://github.com/sparkbox/accessible-components/pull/62/commits/c692e8fcbbada561a630e30c78e7fced57e6ce22